virt/backup_restore/virt-backup-restore-overview.adoc

```diff
@@ -26,6 +26,7 @@ Back up and restore virtual machines by using the xref:../../backup_and_restore/
 [id="additional-resources_virt-backup-restore-overview"]
 == Additional resources
 
+* xref:../../backup_and_restore/application_backup_and_restore/installing/installing-oadp-kubevirt.adoc#installing-oadp-kubevirt[Configuring the {oadp-full} with {VirtProductName}]
```

Comment:
This is ok for now, but my instinct is that this assembly should be removed along with the others, or be simplified so that the link to the `backup_and_restore/application_backup_and_restore/installing/installing-oadp-kubevirt.adoc` assembly is the focal point. Thoughts?

Reply:
I think we need *something* in the virt backup and restore folder to direct users to the installing-oadp-kubevirt.adoc assembly. You could just delete all of the content in this assembly and replace it with something like this (refine as necessary):

```
Back up and restore virtual machines by using the xref:../../backup_and_restore/index.adoc#application-backup-restore-operations-overview[OpenShift API for Data Protection (OADP)]. For more information, see xref:../../backup_and_restore/application_backup_and_restore/installing/installing-oadp-kubevirt.adoc#installing-oadp-kubevirt[Configuring the {oadp-full} with {VirtProductName}].
```

Reply:
I now see that there is also a rosa/dedicated-specific section that has no links in it but does have the steps. We probably should remove those, right? I wonder if the short redirect description I proposed could be included in the ROSA docs (with the links) or if we should just comment this assembly out of those topic maps.

Reply:
@ousleyp - I am confused about this page `virt-backup-restore-overview`. It is an assembly with a procedure in it, the procedure should be in a module. I left it in Virt, but commented it out in the OCD and ROSA topic maps. The xref will be problematic, as OADP docs do not link across.

I have done what I can, please can you have a look and see what you think.

Reply:
I could simply bring across `openshift-docs/backup_and_restore/application_backup_and_restore/installing/installing-oadp-kubevirt.adoc` which puts this procedure into a module, which will then be updated along with the OADP docs

Reply:
So what I would do is something like this - https://75004--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/backup_restore/virt-backup-restore-overview

This assembly includes the modules I created for OADP 

And then do you think we could add `ifdef` statements to remove the problematic links for ROSA and OSD?

WDYT

Reply:
That plan seems like a good compromise, as long as it's not in conflict with the original request to remove the OADP content from the Virt docs. From my perspective, it seems like a more customer-focused solution than completely removing it since we can't provide those links in all distros.

Reply:
As the new doc is basically an assembly with existing modules. It should be easy to keep up to date.